setup.py

```diff
@@ -46,8 +46,9 @@ def get_version():
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     install_requires=[
-        'django>=1.11,<3',
+        'django>=1.11',
```

Comment:
Can you please add `<4`?

Reply:
Would out class this as a patch update or a major update as we're adding support for a new version of Django (from 2 to 3)

Reply:
I'd say new major version

Reply:
amended version to `2.0.0` and limmited support to versions of Django less than 4

Reply:
tests now passing after adding correct `install_requires`